docs/contributing/how-to-contribute.md

```diff
@@ -0,0 +1,23 @@
+---
+title: How to Contribute
+---
+
+We want contributing to Gatsby to be fun, enjoyable, and educational for anyone and everyone.
+
+### Not sure how to start contributing?
+
+If you are worried or don't know where to start, be sure to check out our [where to participate](/contributing/where-to-participate/) page. You can also reach out to [Marcy Sutton (@marcysutton) on Twitter](https://twitter.com/marcysutton) or submit an issue and a maintainer can help give you guidance!
```

Comment:
Might be worth mentioning [pair programming sessions](https://www.gatsbyjs.org/docs/pair-programming/), or otherwise including the core team somehow more obviously in this offer (which we're happy to do so!)

Reply:
Pair programming would be great to call out. I think it's helpful to give a direct point of contact, but I can see that becoming unsustainable. Probably not worth blocking over; we can adjust this later as we learn more about how this section is used.

Reply:
That's an easy addition. I've included it on other pages, it makes sense to include it here too!

Reply:
Updated!